### PR TITLE
add .bz tld parser fix

### PR DESCRIPTION
**Pull request by gremur**
add dedicated parser for .bz tld

### CODE REVIEW

**Comment by gremur** [comment]
Hello, this fix related with following issue: [korall.bz parse error #147](https://github.com/richardpenman/whois/issues/147#issue-1446551321)

File: whois/parser.py [hunk 1]
```diff
@@ -1255,71 +1290,136 @@ def __init__(self, domain, text):
...
-    """Whois parser for .su domains
+    """
+    Whois parser for .su domains
     """
 
     def __init__(self, domain, text):
         WhoisRu.__init__(self, domain, text)
 
 
 class WhoisBz(WhoisRu):
```
Comment (richardpenman) [inline whois/parser.py:1310]: this should inherit from WhoisEntry

File: whois/parser.py [hunk 2]
```diff
@@ -135,6 +135,7 @@ class WhoisEntry(dict):
     yearfirst = False
 
     def __init__(self, domain, text, regex=None):
+        super().__init__()
```
Comment (richardpenman) [inline whois/parser.py:138]: What was the reason to add this?

File: whois/parser.py [hunk 3]
```diff
@@ -389,7 +390,8 @@ def load(domain, text):
 
 
 class WhoisCl(WhoisEntry):
-    """Whois parser for .cl domains."""
+    """
```
Comment (richardpenman) [inline whois/parser.py:393]: Why was a newline added? 
One-line docstrings are standard: https://peps.python.org/pep-0257/

**Review by richardpenman — COMMENTED** [review]
Thanks for PR - added some comments

**Comment by gremur** [comment]
Hello, that was a bad idea to join multiple types of changes in one PR. I'll fix it.

**Pull request closed without merging**